docs/src/param_retrieval.md

```diff
@@ -1,19 +1,58 @@
-# Parameter Dictionaries
+## Basic Parameter Retrieval
 
-Parameters are stored in objects that reflect the model component construction. Definitions should be inserted into the model component source code.
+There are two keys functions for parameter retrieval:
+- `create_toml_dict` constructs a TOML dictionary which stores parameters,
+- `get_parameter_values` retrieves parameters from the TOML dictionary.
 
-## An example from `Thermodynamics.jl`
+To construct a TOML dictionary, pass in the desired floating point type:
+```julia
+import CLIMAParameters as CP
+toml_dict = CP.create_toml_dict(Float64)
+```
+To retrieve parameters, pass in the dictionary and the name(s) of parameters:
```

Comment:
*TOML dictionary

Reply:
*name that matches the defaults list in `ClimaParameters.jl/src/parameters.toml`